app/bolt_listeners.py

```diff
@@ -11,6 +11,8 @@
     OPENAI_TIMEOUT_SECONDS,
     SYSTEM_TEXT,
     TRANSLATE_MARKDOWN,
+    OPENAI_MAX_TOKENS,
+    OPENAI_TEMPERATURE,
```

Comment:
These two env variables may be useful for developers who want to customize how to call OpenAI's API. If you can sent a smaller pull request that includes only these changes, I am happy to have it in this repo.

Reply:
Cool i will make another PR

Reply:
I agree these would be useful.